pinot-core/src/main/java/org/apache/pinot/core/query/scheduler/QueryScheduler.java

```diff
@@ -48,9 +51,13 @@
  */
 public abstract class QueryScheduler {
   private static final Logger LOGGER = LoggerFactory.getLogger(QueryScheduler.class);
+
   private static final String INVALID_NUM_SCANNED = "-1";
   private static final String INVALID_SEGMENTS_COUNT = "-1";
+  private static final String QUERY_LOG_MAX_RATE_KEY = "query_log_max_rate";
```

Comment:
Indicate timeunit. Should the key be "query.log.max.rate" (to conform with other keys we have)?

Reply:
I was following the existing convention of schedulerConfig. Referred `ResourceManager` and it has `query_runner_threads` and `query_worker_threads`. I can do `query.log.max.rate`. Full config name will be `pinot.server.scheduler.query.log.max.rate` since `QueryScheduler` will only get configs with prefix of `pinot.server.scheduler.*`. How do you think?

Reply:
You can keep this for now; we should have a followup to fix it to be consistent.

Reply:
I changed it to `query.log.maxRatePerSecond` to make it consistent with broker. For `query_runner_threads` and `query_worker_threads`, I will address it with a separate pr along with checking other configs.